src/main/java/roboy/dialog/states/expoStates/RoboyQAState.java

```diff
@@ -0,0 +1,218 @@
+package roboy.dialog.states.expoStates;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import roboy.context.Context;
+import roboy.context.contextObjects.IntentValue;
+import roboy.dialog.states.definitions.State;
+import roboy.dialog.states.definitions.StateParameters;
+import roboy.linguistics.Linguistics;
+import roboy.linguistics.Triple;
+import roboy.linguistics.sentenceanalysis.Interpretation;
+import roboy.memory.Neo4jProperty;
+import roboy.memory.Neo4jRelationship;
+import roboy.memory.nodes.Interlocutor;
+import roboy.memory.nodes.MemoryNodeModel;
+import roboy.memory.nodes.Roboy;
+import roboy.talk.PhraseCollection;
+import roboy.util.QAJsonParser;
+import roboy.util.RandomList;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static roboy.memory.Neo4jProperty.*;
+
+public class RoboyQAState extends State {
```

Comment:
please add a comment what this state is doing

Reply:
Added